File: cpp/29c_9A_DieRoll.cpp
#include<bits/stdc++.h>

int main(){
    int y, w, num, den;
    std::cin>>y>>w;
    num = std::max(y, w);
    switch(num){
        case 6 : num = 1, den = 6;
                break;
        case 5 : num = 1, den = 3;
                break;
        case 4 : num = 1, den = 2;
                break;
        case 3 : num = 2, den = 3;
                break;
        case 2 : num = 5, den = 6;
                break;
        case 1 : num = 1, den = 1;
                break;
    }
    std::cout<<num<<"/"<<den;
    return 0;
}

File: python/29c_9A_DieRoll.py
def main():
    y, w = input().split()
    y, w = int(y), int(w)
    num = max(y,w)
    match num:
        case 6 : print("1/6")
        case 5 : print("1/3")
        case 4 : print("1/2")
        case 3 : print("2/3")
        case 2 : print("5/6")
        case 1 : print("1/1")

if __name__ == "__main__":
    main()